src/lib/jsonParseV2/parseStringVector.cpp

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
There is some other PR, about to do a similar change ...
That change would go to master, whereas this PR goes to a feature branch ...
I needed it here, that's why I implemented it, but we need to take care about the PR that goes into master later (if before this feature branch goes into the master branch)

Reply:
The PR you mean is https://github.com/telefonicaid/fiware-orion/pull/3056?

Not sure what do you mean whit the comment... Maybe that master shouldn't be merged into feature/ngsiv2_registrations *before* the present PR merges task/3004_POST_registrations into feature/ngsiv2_registrations? Or maybe the opposite? :)

Reply:
Just warning about a possible future problem, now that I know about it.
How to proceed ... ? 
Well, that depends on the maturity of the branches. Both branches need to be aware, that's all

Reply:
First https://github.com/telefonicaid/fiware-orion/pull/3056 should be merged to master (I think it is only pending on your LGTM ;). Next, feature/ngsiv2_registrations will be updated with master. Next, task/3004_POST_registrations will be updated with feature/ngsiv2_registrations (some conflict may occur).

Makes sense?

Reply:
Sure, the possible problem will be when the feature branch is merged into master

Reply:
We have already done this. Thus, NTC